# PidDebug.msg

# Errors
geometry_msgs/Vector3 error_linear
geometry_msgs/Vector3 error_angular

# Integrals
geometry_msgs/Vector3 integral_linear
geometry_msgs/Vector3 integral_angular

# Derivatives
geometry_msgs/Vector3 derivative_linear
geometry_msgs/Vector3 derivative_angular

# PID Gains
geometry_msgs/Vector3 kp
geometry_msgs/Vector3 ki
geometry_msgs/Vector3 kd
